src/wizeng.main.v3

```diff
@@ -39,82 +39,77 @@ def main(args: Array<string>) -> int {
 		.puts("wizeng: no input files\n")
 		.exit(3);
 
-	if (!Strings.endsWith(path, ".wasm")) return ErrorBuilder.new()
-		.puts(path)
-		.puts(": unknown file format, expected .wasm\n")
-		.exit(4);
-
-	// Parse the binary module
-	var monitors = MonitorOptions.getMonitors();
-	if (monitors.length > 0) Execute.tiering.onMonitorsStart();
-	var result = engine.loadWasmFile(path);
-	var module: Module;
-	match (result) {
-		Ok(m) => {
-			module = m;
-			BasicTracing.instrumentModule(m);
-		}
-		FileNotFound => return ErrorBuilder.new()
-			.puts("wizeng: could not load file ")
+	var all_paths = Vector<string>.new();
+	if (WizengOptions.LINK_MODULES.val != null) {
```

Comment:
As mentioned in person, some adjustments for putting the modules all on the command line are needed here.